NEWS.md

```diff
@@ -1,3 +1,30 @@
+# bbr 1.11.0
+
+## New features and changes
```

Comment:
I think it's probably worth mentioning the name change from 4305fa14 (Remove parenthesis from THETA names in initial_estimates(), gh-674), because that seems like a potential source of breakage.  I'm not too familiar with the other changes coming in.  Any other breaking changes you can think of?

Reply:
That's a fair point actually. Seemed minor but that could definitely impact joining estimates or something. Will revisit the changeset and look for similar items. I was _thinking_ about including the [model status updates](https://github.com/metrumresearchgroup/bbr/pull/693), but @seth127 didn't seem to have a strong opinion there, so I opted to leave them off given the changes are a bit nuanced and wont really affect users in a notable way